.devcontainer/Dockerfile

```diff
@@ -336,17 +336,30 @@ COPY lte/gateway/deploy/roles/magma/files/patches/aioeventlet_fd_exception.patch
 RUN patch -N -s -f /usr/local/lib/python3.8/dist-packages/aioeventlet.py < /tmp/aioeventlet_fd_exception.patch && \
     rm -rf /tmp/*
 
+COPY . ${MAGMA_ROOT}
```

Comment:
Could we copy just the needed files? That might reduce the docker image size, and also express more clearly which files the following steps depend on.

Reply:
Yes, works. Located the relevant sources, as far as i can see.